railties/lib/rails/generators/rails/app/templates/bin/update.tt

```diff
@@ -15,6 +15,11 @@ chdir APP_ROOT do
   puts '== Installing dependencies =='
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
+<% unless options.skip_yarn? -%>
+
+  # Install JavaScript dependencies if using Yarn
+  # system('bin/yarn')
```

Comment:
Is this intended to be commented out?

Reply:
Yes, because even though we install yarn by default, it's not yet every app that actually needs to bundle JS assets every time. If we go all in on Webpacker for JS, that would change.

Reply:
Question is: do we want to merge this change? Is `bin/update` intended to be modified after creation? If not, I don't see the benefit of adding commented out code.

Perhaps `bin/yarn` could run if `package.json` is present?

Reply:
> Is `bin/update` intended to be modified after creation?

Currently at least `bin/setup` is intended to be modified after creation.
For consistency I think we should add this commented out code to `bin/update` or remove the same code from `bin/setup`.
https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/bin/setup.tt#L18-L21

And this code is helpful for us who are using yarn so I think this should be kept.